src/utils/components/SSHSecretModal/components/SSHOptionUseExisting/SSHOptionUseExisting.tsx

```diff
@@ -102,7 +95,13 @@ const SSHOptionUseExisting: FC<SSHOptionUseExistingProps> = ({
     }));
   };
 
-  return !isEmpty(projects) ? (
+  const showNewSecretNameField = namespace
+    ? selectedProject !== namespace
+    : selectedProject !== sshDetails?.sshSecretNamespace;
+
+  return isEmpty(projects) ? (
```

Comment:
can u do isEmpty... return; instead of terenry?

Reply:
Done!